samples/v1/language_entities_text.js

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// DO NOT EDIT! This is a generated sample ("Request",  "language_entities_text")
+
+// sample-metadata:
+//   title: Analyzing Entities
+//   description: Analyzing Entities in a String
+//   usage: node samples/v1/language_entities_text.js [--text_content "California is a state."]
+
+'use strict';
+
+// [START language_entities_text]
+
+const {LanguageServiceClient} = require('@google-cloud/language').v1;
+
+/**
+ * Analyzing Entities in a String
+ *
+ * @param textContent {string} The text content to analyze
+ */
+function sampleAnalyzeEntities(textContent) {
+  const client = new LanguageServiceClient();
+  // const textContent = 'California is a state.';
+
+  // Available types: PLAIN_TEXT, HTML
+  const type = 'PLAIN_TEXT';
+
+  // Optional. If not specified, the language is automatically detected.
+  // For list of supported languages:
+  // https://cloud.google.com/natural-language/docs/languages
+  const language = 'en';
+  const document = {
+    content: textContent,
+    type: type,
+    language: language,
+  };
+
+  // Available values: NONE, UTF8, UTF16, UTF32
+  const encodingType = 'UTF8';
+  const request = {
```

Comment:
This is how we do things today, but using this code review as a chance to point out some thing I don't love in samples :laughing: 

I don't love that we split out `request` separately from the actual operation being performed, it feels more idiomatic that I would simply:

```js
const {entities} = await client.analyzeEntities({
  document,
  encodingType
})
```

:point_up: where, if I was writing TypeScript, I would be getting type checking hints on the parameters being passed in to the API operation.

I'm also advocating that we only return array tupples when it makes sense to do so (@beccasaurus the latter suggestion isn't a blocker for any of this sample generation work).

CC: @xiaozhenliu-gg5, @alexander-fenster, just some thoughts for future breaking versions of the API as we move towards micro-generators.